src/s3/builders/delete_objects.rs

```diff
@@ -303,15 +309,19 @@ where
     }
 }
 
-// endregion: delete-objects
+// endregion: object-stream
 
-// region: remove-objects
+// region: remove-objects-steaming
```

Comment:
Spelling typo. what is this "region" markup needed for?
```suggestion
// region: remove-objects-streaming
```

Reply:
Fixed,  The `// region` with `// endregion` is to create a folding region, and to keep me sane and organized...

![image](https://github.com/user-attachments/assets/58b7a629-8cfc-4a92-95ba-b3efcd228602)